agent/xds/xds_protocol_helpers_test.go

```diff
@@ -155,6 +157,16 @@ func newTestServerScenarioInner(
 		envoy.Close()
 	})
 
+	sink := metrics.NewInmemSink(1*time.Minute, 1*time.Minute)
+	cfg := metrics.DefaultConfig("consul.xds.test")
+	cfg.EnableHostname = false
```

Comment:
```suggestion
	cfg := metrics.DefaultConfig("consul.xds.test")
	cfg.EnableHostname = false
	cfg.EnableRuntimeMetrics = false
```

minor: I'd suggest setting `EnableRuntimeMetrics` to false as well, to prevent the unnecessary background goroutine from starting.

Reply:
:+1: